<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LTIN619452" failures="16" tests="68" name="Chrome_Test" time="213.822" errors="0" timestamp="2025-06-30T12:17:50 IST">
  <testcase classname="tests.TestScripts" name="validateName" time="0.361">
    <failure type="java.lang.AssertionError" message="  Name needs to fails but pass. expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError:   Name needs to fails but pass. expected [false] but found [true]
at tests.TestScripts.validateName(TestScripts.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- validateName -->
  <testcase classname="tests.TestScripts" name="validateName" time="0.423"/>
  <testcase classname="tests.TestScripts" name="validateName" time="0.295"/>
  <testcase classname="tests.TestScripts" name="validateName" time="0.318"/>
  <testcase classname="tests.TestScripts" name="validateName" time="0.315"/>
  <testcase classname="tests.TestScripts" name="validateName" time="0.313"/>
  <testcase classname="tests.TestScripts" name="validateName" time="0.29"/>
  <testcase classname="tests.TestScripts" name="validateAge" time="0.897"/>
  <testcase classname="tests.TestScripts" name="validateAge" time="0.789"/>
  <testcase classname="tests.TestScripts" name="validateAge" time="0.796"/>
  <testcase classname="tests.TestScripts" name="validateAge" time="0.754"/>
  <testcase classname="tests.TestScripts" name="validateAge" time="0.72"/>
  <testcase classname="tests.TestScripts" name="validateAge" time="0.742"/>
  <testcase classname="tests.TestScripts" name="validateAge" time="0.793"/>
  <testcase classname="tests.TestScripts" name="validatePulse" time="1.108"/>
  <testcase classname="tests.TestScripts" name="validatePulse" time="1.049"/>
  <testcase classname="tests.TestScripts" name="validatePulse" time="1.007"/>
  <testcase classname="tests.TestScripts" name="validatePulse" time="0.935"/>
  <testcase classname="tests.TestScripts" name="validatePulse" time="1.088"/>
  <testcase classname="tests.TestScripts" name="validatePulse" time="0.989"/>
  <testcase classname="tests.TestScripts" name="validatePulse" time="1.032"/>
  <testcase classname="tests.TestScripts" name="validateSystolic" time="1.047"/>
  <testcase classname="tests.TestScripts" name="validateSystolic" time="1.186"/>
  <testcase classname="tests.TestScripts" name="validateSystolic" time="1.059"/>
  <testcase classname="tests.TestScripts" name="validateSystolic" time="1.052"/>
  <testcase classname="tests.TestScripts" name="validateSystolic" time="0.985"/>
  <testcase classname="tests.TestScripts" name="validateSystolic" time="0.619">
    <failure type="java.lang.AssertionError" message="130 SystolicBP needs to pass but fails. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 130 SystolicBP needs to pass but fails. expected [true] but found [false]
at tests.TestScripts.validateSystolic(TestScripts.java:69)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- validateSystolic -->
  <testcase classname="tests.TestScripts" name="validateSystolic" time="0.685">
    <failure type="java.lang.AssertionError" message="131 SystolicBP needs to pass but fails. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: 131 SystolicBP needs to pass but fails. expected [true] but found [false]
at tests.TestScripts.validateSystolic(TestScripts.java:69)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- validateSystolic -->
  <testcase classname="tests.TestScripts" name="validateDiastolic" time="1.434"/>
  <testcase classname="tests.TestScripts" name="validateDiastolic" time="1.404"/>
  <testcase classname="tests.TestScripts" name="validateDiastolic" time="1.36"/>
  <testcase classname="tests.TestScripts" name="validateDiastolic" time="1.406"/>
  <testcase classname="tests.TestScripts" name="validateDiastolic" time="1.475">
    <failure type="org.openqa.selenium.UnhandledAlertException" message="unexpected alert open: {Alert text : Provided BP values are not valid. Please Check Once Again}
  (Session info: chrome=137.0.7151.69): Provided BP values are not valid. Please Check Once Again
Build info: version: &amp;apos;4.32.0&amp;apos;, revision: &amp;apos;d17c8aa950&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;21.0.5&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [96b52637b8fc44872cfee65a7bb8f910, findElement {value=bp-diastolic, using=id}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.69, chrome: {chromedriverVersion: 137.0.7151.119 (e0ac9d12dff..., userDataDir: C:\Users\2401015\AppData\Lo...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50802}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50802/devtoo..., se:cdpVersion: 137.0.7151.69, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 96b52637b8fc44872cfee65a7bb8f910">
      <![CDATA[org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : Provided BP values are not valid. Please Check Once Again}
  (Session info: chrome=137.0.7151.69): Provided BP values are not valid. Please Check Once Again
Build info: version: '4.32.0', revision: 'd17c8aa950'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.5'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [96b52637b8fc44872cfee65a7bb8f910, findElement {value=bp-diastolic, using=id}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.69, chrome: {chromedriverVersion: 137.0.7151.119 (e0ac9d12dff..., userDataDir: C:\Users\2401015\AppData\Lo...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50802}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50802/devtoo..., se:cdpVersion: 137.0.7151.69, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 96b52637b8fc44872cfee65a7bb8f910
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:136)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:215)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:368)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:362)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy20.getWrappedElement(Unknown Source)
at org.openqa.selenium.remote.WebElementToJsonConverter.apply(WebElementToJsonConverter.java:48)
at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:1024)
at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:474)
at pages.HomePage.verify(HomePage.java:73)
at pages.HomePage.diastolicBP(HomePage.java:144)
at tests.TestScripts.validateDiastolic(TestScripts.java:77)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- validateDiastolic -->
  <testcase classname="tests.TestScripts" name="validateDiastolic" time="1.347">
    <failure type="java.lang.AssertionError" message="101 DiastolicBP needs to fails but pass. expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: 101 DiastolicBP needs to fails but pass. expected [false] but found [true]
at tests.TestScripts.validateDiastolic(TestScripts.java:77)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- validateDiastolic -->
  <testcase classname="tests.TestScripts" name="validateDiastolic" time="1.401">
    <failure type="java.lang.AssertionError" message="150 DiastolicBP needs to fails but pass. expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: 150 DiastolicBP needs to fails but pass. expected [false] but found [true]
at tests.TestScripts.validateDiastolic(TestScripts.java:77)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- validateDiastolic -->
  <testcase classname="tests.TestScripts" name="validateCalculate" time="0.433"/>
  <testcase classname="tests.TestScripts" name="validateScoresGenerated" time="1.13"/>
  <testcase classname="tests.TestScripts" name="validateScoresGenerated" time="0.974"/>
  <testcase classname="tests.TestScripts" name="validateScoresGenerated" time="1.091"/>
  <testcase classname="tests.TestScripts" name="validateScoresGenerated" time="1.168"/>
  <testcase classname="tests.TestScripts" name="validateScoresGenerated" time="1.085"/>
  <testcase classname="tests.TestScripts" name="validateScoresGenerated" time="0.891">
    <failure type="java.lang.AssertionError" message="Failing to provide score for inputs :56,60-80 BPM,130,85">
      <![CDATA[java.lang.AssertionError: Failing to provide score for inputs :56,60-80 BPM,130,85
at tests.TestScripts.validateScoresGenerated(TestScripts.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- validateScoresGenerated -->
  <testcase classname="tests.TestScripts" name="validateScoresGenerated" time="0.784">
    <failure type="java.lang.AssertionError" message="Failing to provide score for inputs :75,60-80 BPM,131,86">
      <![CDATA[java.lang.AssertionError: Failing to provide score for inputs :75,60-80 BPM,131,86
at tests.TestScripts.validateScoresGenerated(TestScripts.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- validateScoresGenerated -->
  <testcase classname="tests.TestScripts" name="validateScoresGenerated" time="0.816">
    <failure type="java.lang.AssertionError" message="Failing to provide score for inputs :76,81-90 BPM,139,89">
      <![CDATA[java.lang.AssertionError: Failing to provide score for inputs :76,81-90 BPM,139,89
at tests.TestScripts.validateScoresGenerated(TestScripts.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- validateScoresGenerated -->
  <testcase classname="tests.TestScripts" name="validateScoresGenerated" time="0.706">
    <failure type="java.lang.AssertionError" message="Failing to provide score for inputs :1000,81-90 BPM,140,90">
      <![CDATA[java.lang.AssertionError: Failing to provide score for inputs :1000,81-90 BPM,140,90
at tests.TestScripts.validateScoresGenerated(TestScripts.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- validateScoresGenerated -->
  <testcase classname="tests.TestScripts" name="validateScoresGenerated" time="0.739">
    <failure type="java.lang.AssertionError" message="Failing to provide score for inputs :45,91-100 BPM,149,99">
      <![CDATA[java.lang.AssertionError: Failing to provide score for inputs :45,91-100 BPM,149,99
at tests.TestScripts.validateScoresGenerated(TestScripts.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- validateScoresGenerated -->
  <testcase classname="tests.TestScripts" name="validateScoresGenerated" time="0.862"/>
  <testcase classname="tests.TestScripts" name="validateScoresGenerated" time="1.056"/>
  <testcase classname="tests.TestScripts" name="validateScoresGenerated" time="0.784">
    <failure type="java.lang.AssertionError" message="Failing to provide score for inputs :25,60-80 BPM,125,80">
      <![CDATA[java.lang.AssertionError: Failing to provide score for inputs :25,60-80 BPM,125,80
at tests.TestScripts.validateScoresGenerated(TestScripts.java:125)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- validateScoresGenerated -->
  <testcase classname="tests.TestScripts" name="validateScoresGenerated" time="0.872"/>
  <testcase classname="tests.TestScripts" name="alertConsultDoctor" time="16.131">
    <failure type="java.lang.AssertionError" message="Systolic BP : 130  Diastolic BP : 70  Pulse BP :Below 40Alert not generated to consult a doctor">
      <![CDATA[java.lang.AssertionError: Systolic BP : 130  Diastolic BP : 70  Pulse BP :Below 40Alert not generated to consult a doctor
at pages.HomePage.consultDoctor(HomePage.java:236)
at tests.TestScripts.alertConsultDoctor(TestScripts.java:133)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- alertConsultDoctor -->
  <testcase classname="tests.TestScripts" name="alertNotInRange" time="79.822">
    <failure type="java.lang.AssertionError" message="Systolic BP : 130  Diastolic BP : 69  Pulse BP :Below 40 Alert not generated for BP values not in range">
      <![CDATA[java.lang.AssertionError: Systolic BP : 130  Diastolic BP : 69  Pulse BP :Below 40 Alert not generated for BP values not in range
at pages.HomePage.notInRange(HomePage.java:252)
at tests.TestScripts.alertNotInRange(TestScripts.java:139)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- alertNotInRange -->
  <testcase classname="tests.TestScripts" name="scoreCard" time="42.132">
    <failure type="java.lang.AssertionError" message="Systolic BP : 130  Diastolic BP : 70  Pulse BP :50-59 BPM result not generated alert interrupted">
      <![CDATA[java.lang.AssertionError: Systolic BP : 130  Diastolic BP : 70  Pulse BP :50-59 BPM result not generated alert interrupted
at pages.HomePage.success(HomePage.java:272)
at tests.TestScripts.scoreCard(TestScripts.java:146)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- scoreCard -->
  <testcase classname="tests.TestScripts" name="validateHealthRemark" time="0.741"/>
  <testcase classname="tests.TestScripts" name="validateHealthRemark" time="0.866"/>
  <testcase classname="tests.TestScripts" name="validateHealthRemark" time="0.833"/>
  <testcase classname="tests.TestScripts" name="validateHealthRemark" time="0.87"/>
  <testcase classname="tests.TestScripts" name="validateHealthRemark" time="0.945"/>
  <testcase classname="tests.TestScripts" name="validateHealthRemark" time="0.888"/>
  <testcase classname="tests.TestScripts" name="validateHealthRemark" time="0.851"/>
  <testcase classname="tests.TestScripts" name="validateHealthRemark" time="0.913"/>
  <testcase classname="tests.TestScripts" name="validateHealthRemark" time="0.912"/>
  <testcase classname="tests.TestScripts" name="validateHealthRemark" time="0.969"/>
  <testcase classname="tests.TestScripts" name="validateHealthRemark" time="0.856"/>
  <testcase classname="tests.TestScripts" name="validateHealthRemark" time="0.858"/>
  <testcase classname="tests.TestScripts" name="validateHealthRemark" time="0.75"/>
  <testcase classname="tests.TestScripts" name="validateHealthRemark" time="0.787"/>
  <testcase classname="tests.TestScripts" name="validateResetButton" time="0.907">
    <failure type="java.lang.AssertionError" message="Result card should be hidden after reset. expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Result card should be hidden after reset. expected [false] but found [true]
at tests.TestScripts.validateResetButton(TestScripts.java:173)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- validateResetButton -->
</testsuite> <!-- Chrome_Test -->
